manifest.php

```diff
@@ -22,7 +22,7 @@
     'label' => 'Manage test runner plugins',
     'description' =>  "Manage test runner's plugins",
     'license' => 'GPL-2.0',
-    'version' => '2.0.0',
+    'version' => '2.0.1',
```

Comment:
I'd also at least update the feature version and maybe the breaking since the extension won't be backward compatible